executor/executor_test.go

```diff
@@ -1938,6 +1938,7 @@ func (s *testSuite) TestTimestampTimeZone(c *C) {
 	r.Check(testkit.Rows("123381351 1734 2014-03-31 08:57:10 127.0.0.1")) // Cover IndexLookupExec
 
 	// For issue https://github.com/pingcap/tidb/issues/3485
+	tk.MustExec("set time_zone = 'Asia/Shanghai'")
```

Comment:
Why should we add this line? Is there any dst issue in the test case?

Reply:
Please refer to line 1907. It sets timezone with offset -6. If we want to compare it correctly,  we have to set timezone as `Asia/Shanghai`.